scripts/run_pylint.py

```diff
@@ -36,8 +36,9 @@
     'docs/conf.py',
     'setup.py',
 ]
-PRODUCTION_RC = 'pylintrc_default'
-TEST_RC = 'pylintrc_reduced'
+SCRIPTS_DIR = os.path.abspath(os.path.dirname(__file__))
+PRODUCTION_RC = os.path.join(SCRIPTS_DIR, 'pylintrc_default')
+TEST_RC = os.path.join(SCRIPTS_DIR, 'pylintrc_reduced')
```

Comment:
I don't see the move of `pylintrc_reduced` into the `scripts` subdirectory, unlike the move above of `pylintrc_default`.